config/config.go

```diff
@@ -116,14 +112,6 @@ type Limits struct {
 
 // MenderShellConfigFromFile holds the configuration settings read from the config file
 type MenderShellConfigFromFile struct {
-	// ClientProtocol "https"
-	ClientProtocol string
-	// HTTPS client parameters
-	HTTPSClient https.Client `json:"HttpsClient"`
-	// Skip CA certificate validation
-	SkipVerify bool
-	// Path to server SSL certificate
-	ServerCertificate string
```

Comment:
this is a separate ticket, which has nothing to do with MEN-5203. it is removing https support from mender-connect. if we have decided that mender-connect now uses only http it is fine, but otherwise, the ServerCertificate should be provided over D-Bus as ServerURL is. In that case we should keep SkipVerify (or also pass it via D-Bus).

Reply:
It wasn't originally mentioned in MEN-5203, but @lluiscampos and me discussed this on Slack. The reason we want to remove it is that the D-Bus API will never return an https address anymore. It will always be an http link to the local proxy. Hence there is no need for anything certificate related in mender-connect.

Reply:
@merlin-northern I will extend the acceptance criteria on the ticket to cover this. I also made it on a separate commit to leave the discussion open.

Reply:
Ticket updated, commits squashed now.